persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -185,7 +186,7 @@ AND (protocol IS "PROTEUS" OR (protocol IS "MLS" AND mls_group_state IS "ESTABLI
 ORDER BY lastModifiedDate DESC, name COLLATE NOCASE ASC;
 
 selectAllConversations:
-SELECT * FROM ConversationDetails WHERE type IS NOT 'CONNECTION_PENDING' ORDER BY last_modified_date DESC, name ASC;
+SELECT * FROM ConversationDetails WHERE type IS NOT 'CONNECTION_PENDING' AND type IS NOT 'SENT' ORDER BY last_modified_date DESC, name ASC;
```

Comment:
why added this filter?

Reply:
It was for testing purposes but I will revert it